crates/superposition_types/src/lib.rs

```diff
@@ -82,107 +88,6 @@ impl<T> Exp<T> {
     }
 }
 
-macro_rules! impl_try_from_map {
-    ($wrapper:ident, $type:ident, $validate:expr) => {
-        impl TryFrom<Map<String, Value>> for $wrapper<$type> {
-            type Error = String;
-
-            fn try_from(map: Map<String, Value>) -> Result<Self, Self::Error> {
-                Ok(Self($validate(map)?))
-            }
-        }
-
-        impl<'de> Deserialize<'de> for $wrapper<$type> {
-            fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
-            where
-                D: Deserializer<'de>,
-            {
-                let map = Map::<String, Value>::deserialize(deserializer)?;
-                Self::try_from(map).map_err(serde::de::Error::custom)
-            }
-        }
-
-        impl $wrapper<$type> {
-            pub fn try_from_db(map: Map<String, Value>) -> Result<Self, String> {
-                #[cfg(feature = "disable_db_data_validation")]
-                return Ok(Self($type(map)));
-                #[cfg(not(feature = "disable_db_data_validation"))]
-                return Self::try_from(map);
-            }
-        }
-    };
-}
```

Comment:
INFO: Moved to [here](https://github.com/juspay/superposition/pull/264/files#diff-61d7d423a4c1cc86aa26179f4586efff51e9d8dc8a0d414868d820073e622de6R13-R42)